client/src/components/DateTime/style.module.scss

```diff
@@ -0,0 +1,12 @@
+
+.text{
+
+    color: #7d7d7d; 
+    font-family: 'Ropa Sans';
```

Comment:
Can you add `sans-serif` as a fall-back font, use `1.5rem` for the font-size, and remove the blank lines